### PR TITLE
Bob: Changed the param name to further clarify input, references Issu…

### LINKED ISSUE #546: bob: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by atg-abhishek**
Fixes #546

### CODE REVIEW

**Comment by behrtam** [comment]
Thanks for your work! Could you please improve your commit message. Have a look at this great writeup: https://chris.beams.io/posts/git-commit/

**Comment by atg-abhishek** [comment]
Hi! Thanks for the feedback, is there a way that I can directly edit the commit message or would I need to commit again?

**Comment by m-a-ge** [comment]
@atg-abhishek yes, you have to recommit. This guide might be helpful - https://help.github.com/articles/changing-a-commit-message/

File: exercises/bob/bob.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-def hey(string):
+def hey(incoming_words):
```
Comment (m-a-ge) [inline exercises/bob/bob.py]: Maybe just `words` or even `phrase` is a better choice?

Comment (atg-abhishek) [reply exercises/bob/bob.py]: You're right, I agree with phrase, will recommit with that

**Comment by atg-abhishek** [comment]
I've made the change incorporating the comments about the commit message and the param name

**Comment by m-a-ge** [comment]
@atg-abhishek merged, thanks

Just for future reference a guide of how to write good commit messages - https://chris.beams.io/posts/git-commit/#seven-rules

**Pull request merged**